### ISSUE TITLE
Rework deploy process of kernel driver

### ISSUE THREAD

**Issue by lungdear** [OpenSDN-io/community#8]
Rework deploy process of kernel driver for Centos based OS (Rocky, RHEL, ...)
- build it at runtime (deploy stage) instead of pre-build at compile stage
- Ubuntu kernel driver built at run-time
- CentOS/RedHat versions pin kernel version; so built at compile stage instead of run-time

**Comment by randybias** [comment]
Need updates from @Andrey-mp and @tikitavi on current status.

**Comment by Andrey-mp** [comment]
@tikitavi is started with this. I suspect 1-2 weeks for implementation.

**Comment by randybias** [comment]
Updates?

**Comment by Andrey-mp** [comment]
honestly first attempt failed. we tried another solution - it works. but it has limitation for RedHat based OS-es which is not so big and can be handled in future.

**Comment by randybias** [comment]
Is this resolved?  The dependent issue #26 is  closed so I assume it is?

**Comment by Andrey-mp** [comment]
Not yet. @tikitavi successed to build and deploy kernel driver for rocky 9.1
but we have to support all rocky versions now.

**Comment by randybias** [comment]
Updates?

**Comment by tikitavi** [comment]
9.2 is supported
9.3 still has some issues with dhclient I suppose, I'm investigating

**Comment by tikitavi** [comment]
Due to investigations:

The kernel for vrouter on rocky9.3 is compiling and assembling, But another problem arises there - on rocky9 network-scripts package was deprecated, including the ifup and ifdown we used in the agent. Up to 9.2 they nevertheless worked, but in 9.3, they stop working as expected (in particular, canceling the receipt of an address via dhcp for a physical interface is broken).

Closing this task, creating another of reworking vrouter-agent scripts.
